documentation/book/getting-started.adoc

```diff
@@ -34,6 +34,8 @@ For more information about {OpenShiftName} and setting up {OpenShiftName} cluste
 
 include::con-product-downloads.adoc[leveloffset=+1]
 
+include::con-container-images.adoc[leveloffset=+1]
```

Comment:
You are placing this on a very prominent place. But this is something was is very advanced and most users don§t care about it. I do not think this should be in Getting started and definitely not on the beginning.

Reply:
I can drop it in an appendix? What should we do with sections 2.4.2.1, 2.4.2.2 and 3.3.1.5?

Reply:
I think appendix might be too hard to notice. But I can for example see it as 2.10 at the end of the _Getting started_ chapter.

Reply:
I've moved to a section 2.10